pages/common/fvm.md

```diff
@@ -0,0 +1,32 @@
+# fvm
+
+> Flutter version manager.
+> More information: <https://fvm.app/documentation/guides/basic-commands>.
+
+- Install a version of the Flutter SDK. Use without `version` for project settings:
+
+`fvm install {{version}}`
+
+- Set a specific version of Flutter SDK in a project:
+
+`fvm use {{version}} {{options}}`
+
+- Set a global version of the Flutter SDK:
+
+`fvm global {{version}}`
+
+- Delete the FVM cache:
+
+`fvm destroy`
+
+- Remove a specific Flutter SDK:
+
+`fvm remove {{version}}`
+
+- List all versions of the Flutter SDK that have been installed:
```

Comment:
```suggestion
- List all installed versions of the Flutter SDK:
```

Reply:
Good point. Now fixed in the latest commit.